packages/eslint-plugin/src/rules/strict-export-declare-modifiers.ts

```diff
@@ -0,0 +1,193 @@
+import { ESLintUtils } from "@typescript-eslint/utils";
+import * as ts from "typescript";
+import { createRule } from "../util";
+
+const rule = createRule({
```

Comment:
Note that this has to be added to the preset as well in the other file, now that we're doing that.